pkg/reviewapps/permissions.go

```diff
@@ -17,7 +18,12 @@ func canCreate(logger *zap.Logger, appName, token string) (bool, error) {
 		return false, err
 	}
 
-	return k8s.CanDeploy(appName, token), nil
+	canDeploy, err := k8s.CanDeploy(appName, fmt.Sprintf("--token=%s", token))
+	if err != nil {
+		return false, err
+	}
+
+	return canDeploy, nil
```

Comment:
```suggestion
	return k8s.CanDeploy(appName, fmt.Sprintf("--token=%s", token))
```

`k8s.CanDeploy` return values satisfy `canCreate`'s and if `canCreate` errors, it already returns `false, err` so you're just overwriting `canDeploy` on line 23.